calliope/backend/pyomo/model.py

```diff
@@ -118,9 +119,6 @@ def generate_model(model_data):
     if hasattr(backend_model, 'loc_techs_conversion_plus'):
```

Comment:
Hm, given this change: would it make more sense to list all `load_constraints` functions by default, if we are checking within the respective `load_constraints` functions for existence of sets anyway, for ease and consistency? Even automagically load all existing constraint submodules and their load_constraints functions so they don't need explicit listing?

Reply:
See #221